src/dialogs/settingsdialog.ui

```diff
@@ -4756,7 +4766,7 @@ Just test yourself if you get sync conflicts and set a higher value if so.</stri
                 <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
```

Comment:
Can you please not change the default font. 😁  I know, QtCreator tends to do this...

Reply:
Sorry I tried to remove everything, which was not part of adding the checkbox, but it seems that I overlooked something.

Reply:
One of the quirks of QtCreator...